README.md

```diff
@@ -214,6 +214,7 @@ Table of Contents
   * [Efemarai](https://efemarai.com) - Testing and debugging platform for ML models and data. Visualize any computational graph. Free 30 debugging sessions per month for developers.
   * [ExtendsClass](https://extendsclass.com/rest-client-online.html) - Free web-based HTTP client to send HTTP requests.
   * [Export SDK](https://exportsdk.com) - PDF generator API with drag and drop template editor that also provides an SDK and no-code integrations. Free plan comes with 250 pages per month, unlimited users and three templates.
+  * [FormX.ai](https://www.formx.ai) - Data Capture API that uses AI to turn physical documents such as Identity Documents, Receipts, Invoices, into structured digital data (JSON). Free plan includes 100 API calls.
```

Comment:
Afraid to qualify the free tier would need to be fist 100 api calls every month.  Is that how it works?